docker-images/openemr/Dockerfile

```diff
@@ -17,14 +17,15 @@ ENV APACHE_LOG_DIR=/var/log/apache2
 COPY php.ini /etc/php7/php.ini
 COPY openemr.conf /etc/apache2/conf.d/
 #add runner and auto_configure and prevent auto_configure from being run w/o being enabled
-COPY run_openemr.sh .
-COPY auto_configure.php .
-RUN chmod 700 run_openemr.sh \
-    && chmod 000 auto_configure.php \
+COPY run_openemr.sh auto_configure.php utilities/unlock_admin.php /var/www/localhost/htdocs/openemr/
+COPY utilities/unlock_admin.sh /root
```

Comment:
same question as above (since this is specific to lightsail)

Reply:
disregard this post since can just place the unlock_admin.php in this /root/ dir and lock the permissions down, so then safe to include this hook in all dockers.